package.json

```diff
@@ -120,6 +120,7 @@
     "webpack": "^5.28.0",
     "webpack-cli": "^4.6.0",
     "webpack-dev-server": "^3.3.1",
-    "webpack-merge": "^5.1.4"
+    "webpack-merge": "^5.1.4",
+    "wicg-inert": "3.1.1"
```

Comment:
### Todo
We'll want to back this out after the fixes in terra-polyfill and terra-application resolve for wicg-inert@3.1.2